integration-tests/kubernetes/utils.go

```diff
@@ -0,0 +1,19 @@
+package kubernetes
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+// CheckPodsRunning returns true if the number of pods is equal to expected pods and all pods are in running state
+func CheckPodsRunning(pods v1.PodList, expectedPods int) bool {
```

Comment:
any suggestions for improving this? How about creating predicate functions and testing against that ?